pkg/models/accessibility_request.go

```diff
@@ -22,12 +22,12 @@ const (
 type AccessibilityRequest struct {
 	ID             uuid.UUID                           `json:"id"`
 	Name           string                              `json:"name"`
-	IntakeID       uuid.UUID                           `db:"intake_id"`
-	CreatedAt      *time.Time                          `db:"created_at" gqlgen:"submittedAt"`
-	UpdatedAt      *time.Time                          `db:"updated_at"`
-	EUAUserID      string                              `json:"euaUserId" db:"eua_user_id"`
+	IntakeID       uuid.UUID                           `db:"intake_id" json:"intakeID"`
+	CreatedAt      *time.Time                          `db:"created_at" gqlgen:"submittedAt" json:"createdAt"`
+	UpdatedAt      *time.Time                          `db:"updated_at" json:"updatedAt"`
+	EUAUserID      string                              `json:"euaUserID" db:"eua_user_id"`
 	DeletedAt      *time.Time                          `db:"deleted_at" json:"deletedAt"`
-	DeletionReason *AccessibilityRequestDeletionReason `db:"deletion_reason"`
+	DeletionReason *AccessibilityRequestDeletionReason `db:"deletion_reason" json:"deletionReason"`
```

Comment:
Some of the fields were missing JSON keys, so they were updated to have consistent casing.